freqtrade/optimize/hyperopt.py

```diff
@@ -381,6 +383,62 @@ def print_result_table(config: dict, results: list, total_epochs: int, highlight
             )
         print(table)
 
+    @staticmethod
+    def export_csv_file(config: dict, results: list, total_epochs: int, highlight_best: bool,
+                        csv_file: str) -> None:
+        """
+        Log result to csv-file
+        """
+        if not results:
+            return
+
+        # Verification for overwrite
+        if path.isfile(csv_file):
+            logging.error("CSV-File already exists!")
```

Comment:
```suggestion
            logger.error("CSV-File already exists!")
```
'logging' does not print the module issuing the message properly...
same below (check for other appearances)

Reply:
Hmm yes that was a miss.. strange that it worked :-)